src/primitives/transaction.h

```diff
@@ -374,6 +374,14 @@ class OrchardBundle
         }
         inner.reset(bundle);
     }
+
+    bool outputs_enabled() {
+        return orchard_bundle_outputs_enabled(inner.get());
+    }
+
+    bool spends_enabled() {
+        return orchard_bundle_spends_enabled(inner.get());
+    }
```

Comment:
This PR will need to be rebased on master, as `OrchardBundle` moved to `src/primitives/orchard.h`.